src/analysis/utils/cost-utils.ts

```diff
@@ -326,20 +326,15 @@ export const getCostForDisk = (
   toolLabel: ToolLabel
 ): number => {
   let diskCost = 0;
-  const rawPd = persistentDisks && persistentDisks.length && getCurrentPersistentDisk(runtimes, persistentDisks);
-  const curPd =
-    rawPd &&
-    rawPd.diskType &&
-    (isUndecoratedPersistentDisk(rawPd)
-      ? updatePdType(rawPd)
-      : updatePdType({ ...rawPd, diskType: rawPd.diskType.value }));
+  const curPd: PersistentDisk | undefined =
```

Comment:
this file's use of `PersistentDisk`/`DecoratedPersistentDisk` made me cry a bit, but is too large to fix up here